### ISSUE TITLE
Questions about Platform

### ISSUE THREAD

**Issue by rahils-mitre** [dbasch/semantic-search-tweets#1]
I appreciate you sharing this easy to follow example. I have a couple of questions: 
1) is it possible to run this model on Apple Silicon, and if so, how could I go about changing the CUDA dependencies
2) Is there an easy way to find embedding models that run on Apple Silicon(on HuggingFace or elsewhere)?

**Comment by dbasch** [comment]
Yes, simply remove `device=cuda` (it defaults to cpu) and it will work on Apple Silicon. instructor-xl will be a bit slow, but you can use large or base if it's too slow.

https://instructor-embedding.github.io/
